frontend/src/games/sequence-game.tsx

```diff
@@ -0,0 +1,91 @@
+
+import { useEffect, useRef, useState } from 'react';
+
+function delay(ms: number) {
+  return new Promise((resolve) => setTimeout(resolve, ms));
+}
+
+const SequenceGame = () => {
+  const [isGameStarted, setIsGameStarted] = useState(false);
+  const [litUpButtonId, setLitUpButtonId] = useState<number>();
+  const [sequence, setSequence] = useState<(number)[]>([]);
+  const correctClicks = useRef(0);
+
+  useEffect(() => {
+    if (sequence.length > 0) {
+      showSequence().then(() => setLitUpButtonId(undefined));
+    }
+  }, [sequence]);
+  
+  const showSequence = async () => {
+    for (const buttonId of sequence) {
+      setLitUpButtonId(buttonId);
+      await delay(200);
+      setLitUpButtonId(undefined);
+      await delay(100);
+    }
+  }
+  
+  const getSequence = async () => {
+    try {
+      const querySuffix = sequence.length != 0 ? `?sequence=${sequence.toString()}` : ``;
```

Comment:
Sito inline if-else nereikia. Jei ```sequence.length === 0```, tai ```sequence.toString()``` grazins lygiai ta pati, t. y. empty string. And tbf tada kartu nereikia visos sitos eil., nes tada gali fetch'int ```.../getSequence?sequence=${sequence.toString()}...```.

Reply:
As ta darau, kad apskritai omittint query param'us kai sequence tuscias, nes su `.../getSequence?sequence=` man pradeda HTTP 400 (Bad Request) error'a mest, nors ir back end'e yra optional argument, tai nzn, paliksiu kaip yra

Reply:
Pabandziau, sitas veikia puikiai:
```js
const response = await fetch(
  `http://localhost:5252/SequenceGame/getSequence?sequence=${sequence.toString()}`,
);
```